Lib9c/Action/CombinationEquipment3.cs

```diff
@@ -293,7 +293,7 @@ Equipment equipment
                 new EquipmentItemOptionSheet.Row[0];
             try
             {
-                optionRows = optionSelector.SelectV2(subRecipe.MaxOptionLimit);
```

Comment:
is this result of v2 and v3 are same?

Reply:
If the sum of the `option_ratio` parameters in the `EquipmentItemSubRecipeSheet` was not 1, the result of v2 and v3 would have been different. However, the sum of the option ratio parameters in the `EquipmentItemSubRecipeSheet` is 1.